jerry-core/api/jerry.c

```diff
@@ -3088,6 +3089,86 @@ jerry_typedarray_find_by_type (jerry_typedarray_type_t type_name, /**< type of t
 
 #endif /* !CONFIG_DISABLE_ES2015_TYPEDARRAY_BUILTIN */
 
+/**
+ * Calculates the size of the given pattern and creates a RegExp object.
```

Comment:
Strange wording. E.g., see `jerry_create_error` and `jerry_create_error_sz` how variants are usually commented.